providers/vic/operations/pod_creator_test.go

```diff
@@ -87,14 +52,6 @@ func TestNewPodCreator(t *testing.T) {
 	c, err = NewPodCreator(client, store, proxy, nil, persona, portlayer)
 	assert.Nil(t, c, "Expected nil")
 	assert.Equal(t, err, PodCreatorPodCacheError)
```

Comment:
I don't think we should remove the check.  I thought we were changing the error message to be more generic.